### ISSUE TITLE
UI does not build

### ISSUE THREAD

**Issue by borjasotomayor** [uchicago-cs/chiventure#124]
The recently merged code for UI (#100) does not build:

```
$ make
gcc -fPIC -Wall -Wextra -O2 -g -I./include/ -I../common/include/ -MM examples/test_ui.c -MT examples/test_ui.o > examples/test_ui.d
gcc -fPIC -Wall -Wextra -O2 -g -I./include/ -I../common/include/ -MM src/print_functions.c -MT src/print_functions.o > src/print_functions.d
gcc -fPIC -Wall -Wextra -O2 -g -I./include/ -I../common/include/ -MM src/window.c -MT src/window.o > src/window.d
gcc -fPIC -Wall -Wextra -O2 -g -I./include/ -I../common/include/ -MM src/ui.c -MT src/ui.o > src/ui.d
gcc -fPIC -Wall -Wextra -O2 -g -I./include/ -I../common/include/   -c -o src/ui.o src/ui.c
gcc -fPIC -Wall -Wextra -O2 -g -I./include/ -I../common/include/   -c -o src/window.o src/window.c
gcc -fPIC -Wall -Wextra -O2 -g -I./include/ -I../common/include/   -c -o src/print_functions.o src/print_functions.c
make: *** No rule to make target 'src/map.c', needed by 'src/map.o'.  Stop.
```

This needs to be fixed. The top-level Makefile also needs to be updated to build the UI code, so that issues like these can be caught by CI.

**Comment by borjasotomayor** [comment]
I have created PR #125 to start to address this issue.

**Comment by borjasotomayor** [comment]
Fixed by #125
